src/components/PharmacyCard/PharmacyCard.tsx

```diff
@@ -35,6 +35,7 @@ interface PharmacyProps {
   website: string;
   lastUpdated: string;
   vaccineAvailabilities: VaccineAvailabilitiesByDateAndRequirementsInterface;
+  organizationId: number;
```

Comment:
Unless I'm mistaken, organization is optional - as such this should be marked optional with `organizationId?: number`